library/webpack-plugins/config-loader-plugin.js

```diff
@@ -1,16 +1,15 @@
 /*
   This plugin determines when @kaliber/config is imported and pushes the custom config-loader
 */
-module.exports = function configLoaderPlugin() {
+module.exports = function configLoaderPlugin () {
   return {
     apply: compiler => {
       compiler.plugin('normal-module-factory', normalModuleFactory => {
 
         normalModuleFactory.plugin('after-resolve', (data, done) => {
-          const { loaders, rawRequest } = data
-          
-          if (data.rawRequest === '@kaliber/config')
-            loaders.push({ loader: require.resolve('../webpack-loaders/config-loader') })
+          const { loaders } = data
+
+          if (data.rawRequest === '@kaliber/config') { loaders.push({ loader: require.resolve('../webpack-loaders/config-loader') }) }
```

Comment:
Two options:
1. Remove `data.` and put `rawRequest` back in the destructuring thing
2. Remove the destructuring and add `data.` to the loaders